src/main.jsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App.jsx';
+import './index.scss';
 import { Provider } from 'react-redux';
 import store from './redux/store';
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+const root = ReactDOM.createRoot(document.getElementById('root'));
+
+root.render(
   <React.StrictMode>
     <Provider store={store}>
-    <App />
-  </Provider>
-  </React.StrictMode>,
-)
+      <App />
+    </Provider>
```

Comment:
- [ ] still, on your home page, your design is not look like the given design guide, there are spaces between your cards and the cards have borders, this is not so in the given design, the given design cards have alternate background colours, kindly implement all these
your design :point_down: 
![Screenshot from 2023-10-18 19-45-48](https://github.com/richie1988/SuperMario/assets/66097371/3797eee0-863e-4fe4-b60f-9e7dd9ca9558)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/richie1988/SuperMario/assets/66097371/d020eb03-2bc0-4784-8313-63888b7779be)